src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4135,6 +4135,9 @@ export class StudyViewPageStore {
     readonly filteredGenePanelData = remoteData({
         await: () => [this.molecularProfiles, this.samples],
         invoke: async () => {
+            if (_.isEmpty(this.molecularProfiles.result)) {
+                return [];
+            }
```

Comment:
Do you need this? Isn't this method already returning empty array when molecularProfiles is empty?